internal/provider/resources/resource_cluster_test.go

```diff
@@ -65,7 +65,28 @@ func TestAcc_ResourceClusterAwsWithDedicatedDeployments(t *testing.T) {
 						Name:                 awsDeploymentName,
 						Description:          "deployment description",
 						SchedulerSize:        "SMALL",
-					}),
+					}) +
+					fmt.Sprintf(`resource "astro_deployment" "%v_astro" {
+						name = "%v_astro"
+						description = "%s"
+						type = "DEDICATED"
+						cluster_id = %s.id
+						contact_emails = []
+						default_task_pod_cpu = "0.25"
+						default_task_pod_memory = "0.5Gi"
+						executor = "ASTRO"
```

Comment:
is it possible to create clusters with `ASTRO` executor on AZURE or GCP? It would be good to add tests for those as well.

Reply:
Yes, it is. Will add that.